fence/blueprints/data/indexd.py

```diff
@@ -898,6 +931,16 @@ def _generate_google_storage_signed_url(
         )
         return final_url
 
+    def delete(self, bucket, file_id):
+        """
+        Delete data file in Google Cloud Storage. Note that file_id is unused to satisfy
+        polymorphism -- the s3 subclass needs the file_id argument.
+        """
+        with GoogleCloudManager(
+            creds=config["CIRRUS_CFG"]["GOOGLE_STORAGE_CREDS"]
+        ) as gcm:
+            gcm.delete_data_file(bucket, self.file_name())
```

Comment:
```suggestion
            gcm.delete_data_file(bucket, file_id)
```

I think file_id is already the filename here?